### PR TITLE
Magazine List

### LINKED ISSUE #4: Magazine: List of issues
https://www.24life.com/magazine/

There is also a hover effect and mobile version.

### LINKED ISSUE #7: Quote block
maybe no block is needed, if the quote format is exported from Word to HTML. 

https://www.24life.com/magazine/volume-5-issue-12/
Screenshot 2023-07-31 at 10 55 48

### PR DESCRIPTION
**Pull request by rrusher**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #4 
Fix #7

Test URLs:
- Before: https://main--24life--hlxsites.hlx.live/magazine
- After: https://magazine-list--24life--hlxsites.hlx.live/magazine

### CODE REVIEW

File: blocks/magazine-list/magazine-list.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.current-issue,
```
Comment (synox) [inline blocks/magazine-list/magazine-list.css:1]: We need to prefix all these with .magazine-list

File: blocks/quote/quote.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  font-style: italic;
+  letter-spacing: 0.05em;
+  line-height: 1.15;
+  font-weight: 500;
+  padding: 10px 0;
+  position: relative;
+}
+
+.quote .quote-author {
+  color: #999;
```
Comment (synox) [inline blocks/quote/quote.css:26]: Let’s find a similar color in styles.css and var() it.

File: blocks/quote/quote.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  font-style: italic;
+  font-family: var(--body-font-family);
+  font-size: var(--body-font-size-card-header);
+  line-height: 1.45;
+}
+
+.quote .quote-author > div::before {
+  content: "\2014";
```
Comment (synox) [inline blocks/quote/quote.css:34]: What character is this?

File: blocks/magazine-list/magazine-list.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  transform: translateY(0%);
+  opacity: 1;
+}
+
+.other-issues .issue {
+  margin-bottom: 10px;
+}
+
+/* Override section padding */
+main .section {
```
Comment (synox) [inline blocks/magazine-list/magazine-list.css:61]: This is a bit broad. Can we do it with .section.magaine-list-container?

File: blocks/quote/quote.css (hunk 3 above)
Comment (rrusher) [reply blocks/quote/quote.css:34]: em dash

**Review by synox — CHANGES_REQUESTED** [review]
looks great, just check for the extra entry

File: blocks/quote/quote.css (hunk 3 above)
Comment (synox) [reply blocks/quote/quote.css:34]: or if that's not possible just a css comment.

File: blocks/magazine-list/magazine-list.js
[the remark this answers is not in the document]
Comment (rrusher) [reply blocks/magazine-list/magazine-list.js:1]: That was caused by the magazine.docx getting index. It has been removed from the index already.

**Review by synox — APPROVED** [review]

**Pull request merged**